kpTranslator/controllers/query_controller.py

```diff
@@ -15,4 +21,23 @@ def query(request_body):  # noqa: E501
 
     :rtype: Message
     """
-    return 'do some magic!'
+    request = Query.from_dict(request_body)
```

Comment:
You'll want to do validation here.  You could do it by writing a custom decorator.

https://flask.palletsprojects.com/en/1.1.x/patterns/viewdecorators/

Some libraries are:
https://pythonhosted.org/Flask-Inputs/
https://marshmallow.readthedocs.io/en/stable/index.html

Right now if I put in :
```
{"message": {"things": "stuff" }}
```

It'll return a 500 error and the logs will show:
```
kp-translator_1  |   File "/usr/src/app/kpTranslator/controllers/query_controller.py", line 27, in query
kp-translator_1  |     qnodes = query_graph.nodes
kp-translator_1  | AttributeError: 'NoneType' object has no attribute 'nodes'
```

Reply:
Okay, I'll review these links and see if I can implement something. As a temporary workaround, I put in an `if request is None:` statement, but it doesn't seem to work. I guess the error occurs during the `from_dict` call. I tried wrapping it in a try/except statement, but this also doesn't seem to fix it....

Reply:
Actually, this particular error occurs when trying to access query nodes. For now, in the latest commit, I've just added a try/except block to catch this.

Reply:
The try/except catches for me and returns a 200 with the error message `"Query request is invalid!\n'NoneType' object has no attribute 'nodes'"` One of the validation libraries will help with this.  You should return a 422 status code for problems with what the user submitted as input.